mkdocs.yml

```diff
@@ -74,6 +74,7 @@ nav:
           - Template Extensions: 'features/software-templates/template-extensions.md'
```

Comment:
Don't need this change, it's already listed here.

Reply:
Thanks, just assumed and looked where that got me 🤣